src/main/java/com/amazon/ion/impl/macro/SystemMacro.kt

```diff
@@ -26,9 +26,14 @@ enum class SystemMacro(
     IfSingle(-1, IF_SINGLE, listOf(zeroToManyTagged("stream"), zeroToManyTagged("true_branch"), zeroToManyTagged("false_branch"))),
     IfMulti(-1, IF_MULTI, listOf(zeroToManyTagged("stream"), zeroToManyTagged("true_branch"), zeroToManyTagged("false_branch"))),
 
+    // Unnameable, unaddressable macros used for the internals of certain other system macros
+    // TODO: See if we can move these somewhere else so that they are not visible
+    _Private_FlattenStruct(-1, _systemSymbol = null, listOf(zeroToManyTagged("structs"))),
+    _Private_MakeFieldNameAndValue(-1, _systemSymbol = null, listOf(exactlyOneTagged("fieldName"), exactlyOneTagged("value"))),
```

Comment:
🗺️ These have no integer ID and no name, so they are not possible to invoke in an Ion stream or in a user-defined template macro (because the definition of a macro that uses these cannot be serialized). They are only used for supporting `make_struct` and `make_field` respectively.

The methods of `SystemMacro` were also updated to never return these, so the only way to get your hands on it is by explicitly using `SystemMacro._Private_...`. It's not great, but it's good enough for now.